concepticondata/concepticon.tsv

```diff
@@ -1265,7 +1265,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 1264	CHANGE	Spatial relations	To make different.	Action/Process	
 1265	HIGH	Spatial relations	Having relatively great height.	Property	
 1266	BRIDE	Kinship	A woman who is going to get married or has just got married.	Person/Thing	
-1267	NARROW	Spatial relations	Having a small width.	Property	
+1267	NARROW	Spatial relations	(Of a path, cave etc.) having a small width.	Property	
```

Comment:
I remember some examples using objects, maybe add something like "of a cane, straw, etc."

Reply:
But narrow focus more on the idea of something going through something else (water through straw, person through cave or path, missile through path [narrow shot]), and thing is more of width in relation to the common/expected (thin as slim), correct?

Reply:
Yes, that's why I would put cane (and straw, if it's not a drinking straw) to THIN.

Reply:
Ah, I now see what you meant. I thought of a cane: https://en.wikipedia.org/wiki/Walking_stick
and not of a cane: https://en.wikipedia.org/wiki/Cane_(grass)
(albeit the latter is both narrow and thin)